exporter/slowlog.go

```diff
@@ -30,3 +32,36 @@ func (e *Exporter) extractSlowLogMetrics(ch chan<- prometheus.Metric, c redis.Co
 	e.registerConstMetricGauge(ch, "slowlog_last_id", float64(slowlogLastID))
 	e.registerConstMetricGauge(ch, "last_slow_execution_duration_seconds", lastSlowExecutionDurationSeconds)
 }
+
+func (e *Exporter) extractSlowLogDetailsMetrics(ch chan<- prometheus.Metric, c redis.Conn) {
+
+	valuesArr, err := redis.Values(doRedisCmd(c, "SLOWLOG", "GET", "10"))
+	var commandDurationSeconds float64
+
+	if err != nil {
+		return
+	}
+
+	for i := 0; i < len(valuesArr); i++ {
+		if values, err := redis.Values(valuesArr[i], err); err == nil && len(values) > 2 {
```

Comment:
This is not safe. You only check `len(values) > 2` but in line 50 you access the fourth element.

Reply:
Hi @oliver006  , I have removed the blank lines and i have updated the condition on the line 46.